packages/radio-group/src/Radio.tsx

```diff
@@ -62,33 +77,42 @@ const labelColorSet = {
 
 const labelStyle = css`
   display: flex;
-  align-items: flex-start;
-  font-size: 14px;
   line-height: 20px;
   cursor: pointer;
+  align-items: flex-start;
+  font-size: 13px;
+  font-weight: 700;
 `;
 
-// Note colors are not in our palette
 const inputColorSet = {
   [Mode.Light]: css`
     &:checked + ${inputDisplayWrapper.selector} ${inputDisplay.selector} {
-      background-color: #2798bd;
-      border-color: #2798bd;
+      background-color: ${palette.blue.base};
+      border-color: ${palette.blue.base};
```

Comment:
Not sure where to put this comment, but there's a default margin in the `input` element that's not being overridden that gives the component some phantom spacin
<img width="360" alt="Screen Shot 2022-02-25 at 10 46 38 AM" src="https://user-images.githubusercontent.com/2414030/155744749-17345128-2eeb-4297-940e-28d694bd21d4.png">
g